plugins/modules/device_credential_workflow_manager.py

```diff
@@ -1192,416 +1174,416 @@ def get_snmpV3_params(self, snmpV3Details):
         Cisco Catalyst Center based on the provided playbook details.
         Check this API using the check_return_status.
 
         Parameters:
-            CredentialDetails (dict) - Playbook details containing Global Device Credentials.
+            credential_details (dict) - Playbook details containing Global Device Credentials.
 
         Returns:
             self - The current object with updated information.
         """
 
         global_credentials = self.get_global_credentials_params()
-        cliDetails = self.get_cli_credentials(CredentialDetails, global_credentials)
-        snmpV2cReadDetails = self.get_snmpV2cRead_credentials(CredentialDetails, global_credentials)
-        snmpV2cWriteDetails = self.get_snmpV2cWrite_credentials(CredentialDetails,
-                                                                global_credentials)
-        httpsReadDetails = self.get_httpsRead_credentials(CredentialDetails, global_credentials)
-        httpsWriteDetails = self.get_httpsWrite_credentials(CredentialDetails, global_credentials)
-        snmpV3Details = self.get_snmpV3_credentials(CredentialDetails, global_credentials)
-        self.have.update({"globalCredential": {}})
-        if cliDetails:
-            cliCredential = self.get_cli_params(cliDetails)
-            self.have.get("globalCredential").update({"cliCredential": cliCredential})
-        if snmpV2cReadDetails:
-            snmpV2cRead = self.get_snmpV2cRead_params(snmpV2cReadDetails)
-            self.have.get("globalCredential").update({"snmpV2cRead": snmpV2cRead})
-        if snmpV2cWriteDetails:
-            snmpV2cWrite = self.get_snmpV2cWrite_params(snmpV2cWriteDetails)
-            self.have.get("globalCredential").update({"snmpV2cWrite": snmpV2cWrite})
-        if httpsReadDetails:
-            httpsRead = self.get_httpsRead_params(httpsReadDetails)
-            self.have.get("globalCredential").update({"httpsRead": httpsRead})
-        if httpsWriteDetails:
-            httpsWrite = self.get_httpsWrite_params(httpsWriteDetails)
-            self.have.get("globalCredential").update({"httpsWrite": httpsWrite})
-        if snmpV3Details:
-            snmpV3 = self.get_snmpV3_params(snmpV3Details)
-            self.have.get("globalCredential").update({"snmpV3": snmpV3})
+        cli_details = self.get_cli_credentials(credential_details, global_credentials)
+        snmp_v2c_read_details = self.get_snmp_v2c_read_credentials(credential_details, global_credentials)
+        snmp_v2c_write_details = self.get_snmp_v2c_write_credentials(credential_details, global_credentials)
+        https_read_details = self.get_https_read_credentials(credential_details, global_credentials)
+        https_write_details = self.get_https_write_credentials(credential_details, global_credentials)
+        snmp_v3_details = self.get_snmp_v3_credentials(credential_details, global_credentials)
+        self.have.update({"global_credential": {}})
+        if cli_details:
```

Comment:
Add a new line before all the if blocks..

Reply:
Added new line before if, except in case of nested if block